### PR TITLE
feat: add `math/base/special/cosd`

### LINKED ISSUE #33: Implement `cosd`
Compute the cosine of `x` where `x` is in degrees. `@stdlib/math/base/special/cosd`.

### PR DESCRIPTION
**Pull request by saisrikardumpeti**
Resolves #33 

## Description

> What is the purpose of this pull request?

This pull request:

-  Adds functionality to compute `cosd` of any given input (in degree).

| degree | cosd |
| -------- | ------- |
| 90  | 0    |
| 60 |  $\frac{1}{2}$   |
| 45 | $\frac{1}{\sqrt{2}}$    |
| 30 | $\frac{\sqrt{3}}{2}$ |
| 0 | 1 |

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #33
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by saisrikardumpeti** [comment]
@Planeshifter this PR is ready for review.

File: lib/node_modules/@stdlib/math/base/special/cosd/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+*/
+function cosd( x ) {
+	var xRad;
+
+	if ( isInfinite( x ) ) {
+		return NaN;
+	}
+
+	if ( isInteger( ( ( x / 90.0 ) - 1.0 ) / 2.0 ) ) {
+		return 0;
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cosd/lib/main.js]:
```suggestion
		return 0.0;
```

File: lib/node_modules/@stdlib/math/base/special/cosd/test/test.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	t.end();
+});
+
+tape( 'if provided a `NaN`, the function returns `NaN`', function test( t ) {
+	var v = cosd( NaN );
+	t.equal( isnan( v ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function computes the cosine of an angle measured in degrees.(negative values)', function test( t ) {
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cosd/test/test.js]:
```suggestion
tape( 'the function computes the cosine of an angle measured in degrees (negative values)', function test( t ) {
```

File: lib/node_modules/@stdlib/math/base/special/cosd/test/test.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+		} else {
+			delta = abs( y - expected[i] );
+			tol = EPS * abs( expected[i] );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[i]+'. y: '+y+'. E: '+expected[i]+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes the cosine of an angle measured in degrees. (positive values)', function test( t ) {
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cosd/test/test.js]:
```suggestion
tape( 'the function computes the cosine of an angle measured in degrees (positive values)', function test( t ) {
```

**Review by Planeshifter — APPROVED** [review]
Thanks! Will merge once CI has passed.

**Pull request merged**